NOTICE.md

```diff
@@ -0,0 +1,496 @@
+<tr class="b">
+<td>org.springframework</td>
+<td><a class="externalLink" href="https://github.com/spring-projects/spring-framework">spring-webmvc</a></td>
+<td>4.1.4.RELEASE</td>
+<td>jar</td>
+<td><a class="externalLink" href="http://www.apache.org/licenses/LICENSE-2.0.txt">The Apache Software License, Version 2.0</a></td></tr>
+<tr class="a">
+<td>taglibs</td>
+<td>page</td>
+<td>1.0.1</td>
+<td>jar</td>
+<td>-</td></tr>
+<tr class="b">
+<td>taglibs</td>
+<td>request</td>
+<td>1.0.1</td>
+<td>jar</td>
+<td>-</td></tr>
+<tr class="a">
+<td>taglibs</td>
+<td>response</td>
+<td>1.0.1</td>
+<td>jar</td>
+<td>-</td></tr>
+<tr class="b">
+<td>taglibs</td>
+<td>standard</td>
+<td>1.1.2</td>
+<td>jar</td>
+<td>-</td></tr>
+<tr class="a">
+<td>xerces</td>
+<td><a class="externalLink" href="http://xerces.apache.org/xerces2-j">xercesImpl</a></td>
+<td>2.8.0</td>
+<td>jar</td>
+<td><a class="externalLink" href="http://www.apache.org/licenses/LICENSE-2.0.txt">The Apache Software License, Version 2.0</a></td></tr></table></div>
+<div class="section">
+<h3>runtime<a name="runtime"></a></h3><a name="runtime"></a>
+<p>The following dependencies are required to run OpenMRS Core:</p>
+<table border="0" class="bodyTable">
+<tr class="a">
+<th>GroupId</th>
+<th>ArtifactId</th>
+<th>Version</th>
+<th>Type</th>
+<th>License</th></tr>
+<tr class="b">
+<td>mysql</td>
+<td><a class="externalLink" href="http://dev.mysql.com/doc/connector-j/en/">mysql-connector-java</a></td>
+<td>5.1.45</td>
+<td>jar</td>
+<td><a class="externalLink" href="http://www.gnu.org/licenses/old-licenses/gpl-2.0.html">The GNU General Public License, Version 2</a></td></tr>
+<tr class="a">
+<td>org.slf4j</td>
+<td><a class="externalLink" href="http://www.slf4j.org">slf4j-log4j12</a></td>
+<td>1.6.0</td>
+<td>jar</td>
+<td><a class="externalLink" href="http://www.opensource.org/licenses/mit-license.php">MIT License</a></td></tr>
+<tr class="b">
+<td>postgresql</td>
+<td><a class="externalLink" href="http://jdbc.postgresql.org">postgresql</a></td>
+<td>9.0-801.jdbc4</td>
+<td>jar</td>
+<td><a class="externalLink" href="http://jdbc.postgresql.org/license.html">BSD License</a></td></tr></table></div>
+<div class="section">
+<h3>provided<a name="provided"></a></h3><a name="provided"></a>
+<p>The following dependencies can be included when compiling, but should be provided by default when running OpenMRS Core:</p>
+<table border="0" class="bodyTable">
+<tr class="a">
+<th>GroupId</th>
+<th>ArtifactId</th>
+<th>Version</th>
+<th>Type</th>
+<th>License</th></tr>
+<tr class="b">
+<td>javax.servlet</td>
+<td><a class="externalLink" href="http://servlet-spec.java.net">javax.servlet-api</a></td>
+<td>3.0.1</td>
+<td>jar</td>
+<td><a class="externalLink" href="https://glassfish.dev.java.net/nonav/public/CDDL+GPL.html">CDDL + GPLv2 with classpath exception</a></td></tr></table></div></div>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
Why is this here? Can you please make sure your branch is up to date with openmrs/passwordreset and not openmrs/master?